mne/report.py

```diff
@@ -1290,8 +1305,16 @@ class construction.
             info = read_info(self.info_fname, verbose=False)
             sfreq = info['sfreq']
         else:
-            warn('`info_fname` not provided. Cannot render -cov.fif(.gz) and '
-                 '-trans.fif(.gz) files.')
+            # only warn if relevant
+            if any(fname.endswith(('-cov.fif', '-cov.fif.gz'))
+                   for fname in fnames):
+                warn('`info_fname` not provided. Cannot render '
+                     '-cov.fif(.gz) files.')
+            if any(fname.endswith(('-trans.fif', '-trans.fif.gz'))
+                   for fname in fnames):
+                warn('`info_fname` not provided. Cannot render '
+                     '-trans.fif(.gz) files.')
+                raise RuntimeError
```

Comment:
now it's an error while it was just a warning. Can it risk breaking code that was running before?

Reply:
oof yes this is just debugging cruft...